packages/web-components/package.json

```diff
@@ -86,8 +88,10 @@
     "start": "yarn start-storybook -p 6006 --docs",
     "start-storybook": "storybook dev",
     "build-storybook": "storybook build -o ./dist/storybook --docs",
+    "build-harness": "vite build test/harness",
```

Comment:
wondering if you tried https://github.com/storybookjs/builder-vite to get bench data regarding "slow-downs" for your setup ? 

adding another "complexity" for purpose that can be solved via uniform way is mostly always preferable, especially in monorepo context

Reply:
We use `builder-vite` in other Storybook projects, and would love to move to Storybook's `web-components-vite` framework when SB is upgraded to version 8 here. Alternatively, I could rewrite this harness to use Webpack instead of Vite, since that's already present in the monorepo.

For additional context, we decided to stop using Storybook as a harness for Playwright tests in other projects because the preview/iframe page loads too many extra Storybook-specific things, and each test replaces the entire page content after it finishes loading anyway.

Regarding slowdowns: there's some consistency in the flaky test scenarios that occasionally appear, and we suspect that they're related to how the web components are structured, along with how the content is injected into the page. For instance, Playwright's `page.setContent()` method is effectively using `document.write()`, which has varying side effects depending on the browser (this is why our tests weren't run in Webkit until now). `innerHTML` is more consistent, but tests for `radio`/`radio-group`, `menu-item`/`menu`, `accordion-item`/`accordion`, `tab`/`tablist` can all exhibit flakiness, regardless of the method used. This simplified harness makes it easier to continue investigating why these nested component scenarios are flaky.

Reply:
appreciate additional context !

- move to Storybook's web-components-vite: right SB v8  is on our roadmap for this FY
- SB iframes: yeah those are "unfortunate". AFAIR you can open just the frame without getting all that SB "junk" within the test
- no need to webpack your stuff: our plans are to move to vite as well (but for integration scenarios we will need to test against both to guarantee support to our users )

ty!